packages/circuit-ui/components/NotificationBanner/NotificationBanner.stories.tsx

```diff
@@ -28,6 +28,7 @@ export default {
   component: NotificationBanner,
   tags: ['status:stable'],
   parameters: {
+    layout: 'padded',
```

Comment:
A downside of using container queries is that the parent element is no longer sized based on its children since that could cause an infinite loop. 

That's why I had to switch to the "padded" layout here, since the component disappears using the default "centered" layout.